etc/kayobe/kolla.yml

```diff
@@ -177,6 +177,7 @@ kolla_overcloud_inventory_custom_components: "{{ lookup('template', kayobe_confi
 
 # Custom overcloud inventory containing a mapping from services to components.
 #kolla_overcloud_inventory_custom_services:
+kolla_overcloud_inventory_custom_services: "{{ lookup('template', kayobe_config_path ~ '/kolla/inventory/overcloud-services.j2') }}"
```

Comment:
See other comment - I think you can get away without this.  Check with @jovial ?